lib/lighthouse/benefits_claims/configuration.rb

```diff
@@ -13,7 +13,7 @@ module BenefitsClaims
   class Configuration < Common::Client::Configuration::REST
     self.read_timeout = Settings.lighthouse.benefits_claims.timeout || 20
 
-    API_SCOPES = %w[system/claim.read system/claim.write].freeze
+    API_SCOPES = %w[system/claim.read system/claim.write system/526-pdf.override].freeze
```

Comment:
verified that staging and production Lighthouse benefits claims client ids can generate an auth token with these scopes listed
references: https://dsva.slack.com/archives/C02CQP3RFFX/p1713469011121339